cibuildwheel/util.py

```diff
@@ -255,19 +270,13 @@ def detect_ci_provider() -> Optional[CIProvider]:
 
 PRETTY_NAMES = {'linux': 'Linux', 'macos': 'macOS', 'windows': 'Windows'}
 
-ALLOWED_ARCHITECTURES = {
-    'linux': {Architecture.x86_64, Architecture.i686, Architecture.aarch64, Architecture.ppc64le, Architecture.s390x},
-    'macos': {Architecture.x86_64},
-    'windows': {Architecture.AMD64, Architecture.x86},
-}
-
 
 def allowed_architectures_check(
     name: PlatformName,
```

Comment:
Might have missed this first time around, but generally this variable should be called 'platform'